lib/src/delegates/asset_picker_text_delegate.dart

```diff
@@ -11,6 +11,7 @@ import 'package:photo_manager/photo_manager.dart' show AssetType;
 /// All text delegates.
 const assetPickerTextDelegates = <AssetPickerTextDelegate>[
   AssetPickerTextDelegate(),
+  TraditionalChineseAssetPickerTextDelegate(),
```

Comment:
Maybe put this at the bottom? The order doesn't really matter.

Reply:
I sorted the list of delegates alphabetically, except for the default one (AssetPickerTextDelegate), to make it easier for users to find specific delegate.